doc/rules/language_construct/silenced_deprecation_error.rst

```diff
@@ -0,0 +1,27 @@
+===================================
+Rule ``silenced_deprecation_error``
+===================================
+
+.. warning:: This rule is deprecated and will be removed on next major version.
+
+   You should use ``error_suppression`` instead.
+
+Ensures deprecation notices are silenced.
+
+.. warning:: Using this rule is risky.
+
+   Silencing of deprecation errors might cause changes to code behaviour.
+
+Samples
```

Comment:
Maybe just call them `Examples`? It's a bit strange to have a heading called `Samples`, and then the enumeration calling them `Example #X.`

Reply:
Done.